SND/EmulsionTarget/Target.cxx

```diff
@@ -677,40 +512,13 @@ Bool_t  Target::ProcessHits(FairVolume* vol)
 }
 
 
-void Target::DecodeBrickID(Int_t detID, Int_t &NWall, Int_t &NRow, Int_t &NColumn, Int_t &NPlate, Bool_t &EmCES, Bool_t &EmBrick, Bool_t &EmTop)
+void Target::DecodeBrickID(Int_t detID, Int_t &NWall, Int_t &NRow, Int_t &NColumn, Int_t &NPlate, Bool_t &EmTop)
```

Comment:
Same comments as for the SST decoding apply here, we should make use of the possibilities modern C++ provides, rather than doing everything via argument mutation.

See #552